embeddable-say-editor/app/components/simple-editor.hbs

```diff
@@ -10,10 +10,11 @@
   {{#if this.initCompleted}}
     <EditorContainer
       @editorOptions={{hash
-        showRdfa='true'
-        showRdfaHighlight='true'
-        showRdfaHover='true'
-        showPaper='true'
+        showRdfa=false
+        editRdfa=true
```

Comment:
I think since we're not actually enabling any of the RDFa editing tools, we want to have `showRdfa=true` and `editRdfa=false`. Otherwise, we get the squares outlining nodes containing RDFa but no indication of what they signify.